submissions/nallikaea/js-dom/index.js

```diff
@@ -0,0 +1,143 @@
+  },
+  {
+    name: 'Павутинник фіолетовий',
+    imgSrc: './img/pautinnik.jpg',
+    description: 'Не дивлячись на дивний колір - їстівний гриб з роду Павутинник. Занесен до Червоної книги, тому краще не брати. Також можна сплутати з отруйними павутинниками.',
+    receptName: 'Мариновані павутинники',
+    ingredients: 'павутинники 1кг, сіль 2ст/л, цукор 2 ст/л, оцет 2 ст/л, запашний перець горошком 5шт, лист лавра 1шт, часник 5 зубків.',
+    recept:'До води додати спеції, закіп\'ятити і занурити туди підготовлені і відварені гриби, потім тримати на вогні ще 20 хвилин. Розкласти плоди по банкам і долити до верху маринад. Стерилізувати протягом 15 хвилин і закатати кришками.'
+  },
+  {
+    name: 'Польський гриб',
+    imgSrc: './img/polskiy.jpg',
+    description: 'Дуже добрий їстівний гриб. В різних системах класифікації відносяться до родів Боровик або Моховик .',
+    receptName: 'Листкові булочки з грибами',
+    ingredients: 'листкове тісто 1уп, польські гриби 0,25кг, цибулина 1шт, сметана 3 ст/л, жовток 1шт, борошно 1ст/л, сіль, перець.',
+    recept: 'Дрібно нарізати цибулю і відварені гриби. Обсмажити до напівготовності з сіллю і перцем. Додати сметану і борошно, добре перемішати. Тісто нарізати квадратиками і злегка розкатати (10х10см). На середину квадратика покласти начинку. Скласти по діагоналі і защипи по краях виделкою. Пиріжок проколоти виделкою та змастити жовтком. Випікати 20 хвилин при температурі 200 градусів.'
+  },
+  {
+    name: 'Підосиковик',
+    imgSrc: './img/podosinovik.jpg',
+    description:
+      'Вид базидіомікотових грибів родини Болетові. Їстівний смачний гриб.',
+    receptName:'Грибний суп з підосиковиків',
+    ingredients: 'Підосиковики 4шт, картопля - 3шт, цибуля 1шт, морква 1шт, сіль, перець чорний горошком, лавровий лист 2шт. ',
+    recept:'У каструлю влийте воду, посоліть, додайте гриби, спеції і варіть на середньому вогні 20 хвилин. Знімайте пінку шумівкою. Цибулю і моркву наріжте кубиками та обсмажте до золотистого кольору. Картоплю наріжте і додайте в грибний бульйон. Через кілька хвилин додайте смажені моркву і цибулю та варіть до готовності. Подавайте зі сметаною.'
+  }
+];
+
+const DOC = document;
+
+const container = DOC.createElement('div');
+container.classList.add('container');
+DOC.body.appendChild(container);
+
+const dictionaryName = DOC.createElement('h1');
+dictionaryName.classList.add('dictionary-name');
+dictionaryName.textContent = 'Мої знахідки';
+container.appendChild(dictionaryName);
+
+const divMain = DOC.createElement('div');
+divMain.classList.add('main');
+container.appendChild(divMain);
+
+const divMenu = DOC.createElement('div');
+divMenu.classList.add('menu');
+divMain.appendChild(divMenu);
+
+const navigation = DOC.createElement('ul');
```

Comment:
The same here. You can move creation of this element to your html file and query it by `DOC.querySelector('.navigation')` for generating your menu